vscode/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "wdl",
+  "displayName": "WDL Syntax Highlighter",
+  "description": "Official Syntax Highlighter for the Workflow Definition Language",
+  "version": "0.0.1",
```

Comment:
ToL (thinking out loud, ie feel free to decide you disagree): I'm not a huge fan of "never quite 1.0" versions. If we think this is good enough to install we can probably just call it 1.0 and increment from there

Reply:
I agree with you. Changing to 1.0.0.